geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateDefinedIndexesCommand.java

```diff
@@ -61,7 +62,7 @@ public Result createDefinedIndexes(
           help = CliStrings.CREATE_DEFINED_INDEXES__GROUP__HELP) final String[] group) {
 
     Result result;
-    AtomicReference<XmlEntity> xmlEntity = new AtomicReference<>();
+    List<XmlEntity> xmlEntities = Collections.synchronizedList(new ArrayList<XmlEntity>());
```

Comment:
Does this have to be a synchronizedList?

Reply:
I guess I used the `Collections.synchronizedLis` because the old version had an `AtomicReference`. Thanks for pointing this out anyway, it really doesn't need to be synchronized.